package.json

```diff
@@ -118,6 +118,7 @@
     "http-server": "^0.11.1",
     "husky": "^3.1.0",
     "lodash": "^4.17.15",
+    "long": "^4.0.0",
```

Comment:
accidental?

Reply:
At a point I was getting a runtime failure because this was missing. Maybe it's not needed any more

Reply:
Yea we used it for hashing before but that moved to WebAssembly

Reply:
Removed